packages/components/src/form/RadioGroup/RadioGroup.styles.js

```diff
@@ -48,6 +51,7 @@ const RadioGroupStyles = createStyles(
       },
       radio: {
         pointerEvents: 'none',
+        paddingRight: minWidth && 0,
```

Comment:
It's hard to read at first look, maybe do you prefer this?

```suggestion
        paddingRight: minWidth ? 0 : undefined,
```